grids.h warnings  can be eliminated by using -std=c++11

the compilation of grids.h  gives 5 warnings of the form
```
/home/hines/neuron/temp/src/nrnpython/grids.h:164:27: warning: non-static data member initializers only available with ‘-std=c++11’ or ‘-std=gnu++11’
```
One way to avoid these is to modify nrn/CMakeLists.txt from
set(CMAKE_CXX_STANDARD 98)
to
set(CMAKE_CXX_STANDARD 11)
although, since it is just a warning, can we turn off
set(CMAKE_CXX_STANDARD_REQUIRED ON)
Or is 11 now universally available?
Or should grids.h be reimplemented so that it is warning free with 98?

--- a/src/nrnpython/grids.h
+++ b/src/nrnpython/grids.h
@@ -161,7 +161,7 @@ class Grid_node {
     int ics_num_segs;
 
     int insert(int grid_list_index);
-    int node_flux_count = 0;
+    int node_flux_count;
     long * node_flux_idx;
     double * node_flux_scale;
     PyObject ** node_flux_src;

